### DISCUSSION TITLE
Trigger redeployment despite no changes to the stack

### DISCUSSION THREAD

**Discussion by hnorth99** [aws/aws-cdk#28966] [Q&A] [answered]
Is there a way I can trigger my stack to go through the redeployment process despite not making changes to the stack?

For context, I have a stack deployment that triggers secondary actions that I am trying to test out. If cdk doesn't support this, what would be the recommend/easiest way for me to generate this desired behavior?

**Comment by richardgroves** [comment, marked as answer]
This might help:

-f, --force                Always deploy stack even if templates are identical
                                                      [boolean] [default: false]
